frappe/utils/change_log.py

```diff
@@ -103,6 +103,10 @@ def get_versions():
 			branch_version = app_hooks.get('{0}_version'.format(versions[app]['branch']))
 			if branch_version:
 				versions[app]['branch_version'] = branch_version[0] + ' ({0})'.format(get_app_last_commit_ref(app))
+			else:
```

Comment:
beta should be treated like develop. Why do we need this special code?

Reply:
there is no beta branch, branch will be v11 or something...

the current code pulls version based on branch